pkgs/by-name/sn/snander/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  versionCheckHook,
+  nix-update-script,
+  libusb1,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "snander";
+  version = "1.7.8";
+
+  src = fetchFromGitHub {
+    owner = "Droid-Max";
+    repo = "SNANDer";
+    tag = "v.${finalAttrs.version}";
+    hash = "sha256-YsNWjlbUTAURD6eOt8ZVDBGMZT4w0hGFO6N42CdX2XA=";
+  };
+
+  buildInputs = [ libusb1 ];
+
+  makeFlags = [
+    "-C"
+    "src"
+  ];
+
+  installFlags = [
+    "DESTDIR=${placeholder "out"}"
+    "PREFIX="
+  ];
+
+  doInstallCheck = true;
+  nativeInstallCheckInputs = [ versionCheckHook ];
+  versionCheckProgramArg = "-h";
+
+  passthru.updateScript = nix-update-script {
+    extraArgs = [
+      "--version-regex"
+      "^v\.(\d+\.\d+\.\d+)$"
```

If you use double quotes, you’ll have to escape the backslash character with another one. As this quickly becomes unreadable, I suggest to use [double single quotes](https://docs.lix.systems/manual/lix/stable/language/values.html#type-string) instead for this kind of regex.

```suggestion
      ''^v\.(\d+\.\d+\.\d+)$''
```

---

I amended my personal fork's commit and force pushed it. The change is reflected in my personal repo, but the change didn't seem to propagate to this PR. Any suggestions?